test/suites/dev-tanssi-relay/external-validators-rewards/test_external_validator_reward_candidates.ts

```diff
@@ -0,0 +1,68 @@
+import "@tanssi/api-augment";
+import { describeSuite, customDevRpcRequest, expect, beforeAll } from "@moonwall/cli";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { jumpToSession } from "util/block";
+
+describeSuite({
+    id: "DTR1601",
+    title: "Paras inherent tests",
+    foundationMethods: "dev",
+
+    testCases: ({ it, context }) => {
+        let polkadotJs: ApiPromise;
+
+        beforeAll(async () => {
+            polkadotJs = context.polkadotJs();
+        });
+
+        it({
+            id: "E01",
+            title: "para candidates should trigger reward info",
+            test: async function () {
+                const keyring = new Keyring({ type: "sr25519" });
+                const aliceStash = keyring.addFromUri("//Alice//stash");
+                await context.createBlock();
+                // Send RPC call to enable para inherent candidate generation
+                await customDevRpcRequest("mock_enableParaInherentCandidate", []);
+                // Since collators are not assigned until session 2, we need to go till session 2 to actually see heads being injected
+                await jumpToSession(context, 3);
+                await context.createBlock();
+
+                // we are still in era 0
+                const validatorRewards = await context
+                    .polkadotJs()
+                    .query.externalValidatorsRewards.rewardPointsForEra(0);
+                const totalRewards = validatorRewards.total.toBigInt();
+
+                // Validators get 20 points for creating a block, so if they included a candidate, they will get more than 20
+                expect(totalRewards).to.be.greaterThan(20n);
```

Comment:
wait they get 20 per candidate. So you will always have higher than 20?

Reply:
I guess if so, you need to check the rewards `totalRewards > num_blocks*20`

Reply:
*fixed*